### DISCUSSION TITLE
Planned date for 2.0.0?

### DISCUSSION THREAD

**Discussion by stockiNail** [chartjs/chartjs-plugin-datalabels#242] [Q&A] [answered]
I would like to know when the plugin could be released in a stable version (no beta or rc) for Chart.js 3.x.
In some environments it's not allowed to use beta or release candidate.

**Comment by simonbrunel** [comment]
No "planned date": there is still one blocking issue (#233) as listed in the remaining TODOs in [this conversation](https://github.com/chartjs/chartjs-plugin-datalabels/discussions/213).

**Reply by stockiNail** [reply]
Thank you very much for feedback @simonbrunel 

I have seen the issue and it's very similar to the issue that I raised in the PR to port the plugin to CHART.JS 3, where I was using floating bar, with 2 labels. 
If I remember well, it was related to dirty flag calculated at the beginning of drawing.

**Comment by simonbrunel** [comment, marked as answer]
Version 2.0.0 of this plugin has been [released](https://github.com/chartjs/chartjs-plugin-datalabels/releases/tag/v2.0.0) today!

**Reply by stockiNail** [reply]
I see, THANK YOUU!

**Reply by stockiNail** [reply]
I'm gonna add the dependency to new version of my lib (in the current one we have decided to release it with rc-1)
